assets/css/style.css

```diff
@@ -190,7 +194,8 @@ a {
   display: none;
 }
 
-.header-menu-links button {
+.header-menu-links button,
+.works-popup-container button {
   width: 40px;
```

Comment:
[Optional]

kindly work on the mobile view of your popup modal as it seems to be having some scrolling issues hence the `close` button doesn't display on the screen. It was supposed to be a mandatory requirement but I think it was omitted by the previous reviewer so please fix it 😄

Your design : 
![image](https://github.com/mahammad-mostafa/personal-portfolio-site/assets/20537712/5b701bc0-871c-41ec-b974-7aebe39efc5f)

Reply:
Thanks a lot @usorfaitheloho for your thorough review!
I have fixed the scrolling issue successfully 👌🏻